AISKU/scripts/setActiveCdnVersion.ps1

```diff
@@ -664,7 +664,8 @@ if ($container -eq "beta" -or $container -eq "next") {
 if ($files.ContainsKey($activeVersion) -ne $true) {
     Log-Failure "Version [$activeVersion] does not appear to be deployed to [$container]"
 } elseif ($files[$activeVersion].Count -ne 4 -and # Prior to 2.5.8
-        $files[$activeVersion].Count -ne 8) {     # Since 2.5.8
+        $files[$activeVersion].Count -ne 8 -and   # Since 2.5.8
+        $files[$activeVersion].Count -ne 12) {    # Since 2.5.8
```

Comment:
While we have already defined the "label" for 2.5.8, we are planning on sneaking this into the 2.5.8 release as it's only changing whats deployed and not any code.

Reply:
The only specific issue is that this new format won't appear in the NPM package until the next release version bump